classes/Hook.php

```diff
@@ -683,6 +683,8 @@ public static function exec($hook_name, $hook_args = array(), $id_module = null,
                     $display = Hook::coreCallHook($moduleInstance, 'hook'.$hook_name, $hook_args);
                 } elseif ($hook_retro_callable) {
                     $display = Hook::coreCallHook($moduleInstance, 'hook'.$retro_hook_name, $hook_args);
+                } else {
+                    continue;
```

Comment:
I was talking about this:
![variablemightnotbedefined](https://cloud.githubusercontent.com/assets/6775736/18587537/decbf6dc-7c22-11e6-8a60-0c192183f19c.png)

I'd rather have all code inspectable and approved by the IDE.